husky_extensions/ckeditor-extension.js

```diff
@@ -23,7 +23,7 @@
         }
     });
 
-    define(['ckeditor', 'jqueryAdapter'], function() {
+    define(['underscore', 'ckeditor', 'jqueryAdapter'], function(_) {
```

Comment:
we need it to be sure that it is loaded by require. this scripts will add some global vars and constructor's we need.